releases/release-1.31/release-notes/release-notes-draft.json

```
@@ -3021,7 +3103,7 @@
   "125176": {
     "commit": "f44bb5e6e58c315b62c79bbc20814e84eb002e00",
     "text": "```\n\n#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:",
-    "markdown": "```\n  \n  #### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.: ([#125176](https://github.com/kubernetes/kubernetes/pull/125176), [@mauri870](https://github.com/mauri870)) [SIG Testing]",
+    "markdown": "```\n  \n  #### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.: ([#125176](https://github.com/kubernetes/kubernetes/pull/125176), [@mauri870](https://github.com/mauri870))",
```

```suggestion
    "markdown": "```\n  \n  #### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.: ([#125176](https://github.com/kubernetes/kubernetes/pull/125176), [@mauri870](https://github.com/mauri870)) [SIG Testing]",
```